blog/_posts/2016-10-19-scalajs-bundler.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+post-type: blog
+by: Julien Richard-Foy
+title: "scalajs-bundler has been released"
+---
+
+This week we have released a first version of [`scalajs-bundler`](https://scalacenter.github.io/scalajs-bundler), a tool aiming to provide an integrated developer experience for
```

Comment:
What about "This week we release a first version of the [`scalajs-bundler`](https://scalacenter.github.io/scalajs-bundler), a tool that improves the use of [npm]() packages for [Scala.js]() developers." ?